<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>565</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>565</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>12515125</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#1a1a1a;</string>
   </property>
   <widget class="QLineEdit" name="input_number_of_skin">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>400</y>
      <width>530</width>
      <height>60</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> .QLineEdit {
	background-color:#1a1a1a;
	border-radius:10px;
	border:3px solid #4f4f4f;
	color:#ffffff;
	font-family:Trebuchet MS;
	font-size:17px;
}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>What skin do you want to buy? (Just enter the number)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buy_button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>475</y>
      <width>210</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> .QPushButton{
	background-color:#1a1a1a;
	border-radius:10px;
	border:3px solid #4f4f4f;
	color:#ffffff;
	font-family:Trebuchet MS;
	font-size:17px;
}
.QPushButton:hover{
 	background-color:#272727;
	color:#095296;
 }
.QPushButton:pressed {
	border:2px solid #095296;
}</string>
    </property>
    <property name="text">
     <string>Buy</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pprint_text">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>90</y>
      <width>530</width>
      <height>290</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"> .QTextBrowser{
	background-color:#1a1a1a;
	border-radius:10px;
	border:3px solid #4f4f4f;
	color:#ffffff;
	font-family:Trebuchet MS;
	font-size:15px;
}
</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Trebuchet MS'; font-size:15px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_snake_shop_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>481</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#1a1a1a;
	border-radius:10px;
	border:3px solid #4f4f4f;
	color:#ffffff;
	font-family:Trebuchet MS;
	font-size:17px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Snake shop&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
